training/distributed_training/pytorch/model_parallel_v2/README.md

```diff
@@ -0,0 +1,9 @@
+## Sagemaker Model Parallelism
+
+This directory contains example scripts to train or fine-tune large scale models,
+with the Sagemaker distributed model parallelism library.
+When using one of the ipynb notebooks within the folders of this directory please 
+make sure to use the `./shared-scripts/` directory as the source directory when submitting a job.
+
+For example, if one wanted to submit a llama finetune job on Sagemaker using the `/llama_v2/smp-finetuning-llama-fsdp-tp.ipynb`
```

Comment:
Extend this example as if they can just copy paste.
Also mentioned where they will run these notebook? 1) local machine 2) sagemaker notebook instance or something else.

Reply:
Added in a sentence about copying their desired notebook to the shared-scripts folder and suggested running in a Sagemaker notebook.